src/EditorFeatures/CSharpTest/ChangeSignature/ChangeSignature_CheckAllSignatureChanges.cs

```diff
@@ -10,6 +10,21 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ChangeSignature
 {
     public partial class ChangeSignatureTests : AbstractChangeSignatureTests
     {
+        [WorkItem(8333, "https://github.com/dotnet/roslyn/issues/8333")]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.ChangeSignature)]
+        public async Task TestDoNotShowSignatureChangeForExpressionBody()
+        {
+            var markup = @"
+class Point
+{
+    int x; int y;
+    public Point(int x, int y) { }
+    public Point Move1(int dx, int dy) [|=>|] new Point(x + dx, y + dy);
```

Comment:
`[| |]` notation marks the `Span` being selected. `$$` marks the caret position.

Change signature can be invoked as long as the caret is on a method signature. I doubt that it would show up for any selection and that could be the reason why this is passing. In other words, I think we're not testing the intended scenario here..

I think changing it to something like `public Point Move1(int dx, int dy) => new $$Point(x + dx, y + dy);` would be sufficient.

I do not think these nuances ([||] vs $$) are documented, so, I apologize for the inconvenience here.